pkg/cli/demo.go

```diff
@@ -41,14 +44,24 @@ interactive SQL prompt to it. Various datasets are available to be preloaded as
 subcommands: e.g. "cockroach demo startrek". See --help for a full list.
 
 By default, the 'movr' dataset is pre-loaded. You can also use --empty
-to avoid pre-loading a dataset.`,
+to avoid pre-loading a dataset.
+
+cockroach demo attempts to obtain a temporary enterprise license for demoing
```

Comment:
_attempts to connect to a Cockroach Labs server to obtain a temporary enterprise license_ 

(otherwise it's not clear there's a remote roundtrip available.)